device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D40C8S8/hwsku.json

```diff
@@ -0,0 +1,178 @@
+{
+    "interfaces": {
+        "Ethernet0": {
+            "default_brkout_mode": "1x10G"
+        }, 
+        "Ethernet1": {
+            "default_brkout_mode": "1x10G"
+        }, 
+        "Ethernet2": {
+            "default_brkout_mode": "1x10G"
+        }, 
+        "Ethernet3": {
+            "default_brkout_mode": "1x10G"
+        }, 
+        "Ethernet4": {
```

Comment:
Is Ethernet4 blocked? Is it the right way to represent blocked port?

Reply:
Ethernet 4 is blocked. If I remove Ethernet 4 , it will lead to an error when generating the configuration out of the SKU.
ANyway, I will re-check if there is a way to mark this port as blocked.

Reply:
@qiluo-msft  seeking and I didn't find a way to mark the port as blocked on hwsku.json file. removing the port completly will lead to error when enforcing this SKU,